packages/react-core/src/layouts/Toolbar/ToolbarSection.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/layouts/Toolbar/styles.css';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Anything that can be rendered as toolbar section */
+  children: PropTypes.node,
+  /** Classes applied to toolbar section */
+  className: PropTypes.string
+};
+
+const defaultProps = {
+  children: null,
+  className: null
+};
+
+const ToolbarSection = ({ ariaLabel, children, className, ...props }) => (
```

Comment:
remove ariaLabel here

Reply:
We should make this similar to how it is handled in TextInput/TextArea components with the custom console warning.